src/cli.ts

```diff
@@ -1,10 +1,23 @@
-const yargs = require('yargs');
-const builders = require('./builders');
-const { assignGlobals, sanitizeInputs } = require('./utils');
+import yargs from 'yargs/yargs';
+import { assignGlobals, sanitizeInputs } from './utils';
 
-module.exports = function cli(commands) {
-  // eslint-disable-next-line no-unused-expressions
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace NodeJS {
+    interface Global {
+      snaps: {
+        verboseErrors: boolean;
+        suppressWarnings: boolean;
+        isWatching: boolean;
+      };
+    }
+  }
+}
```

Comment:
It's only declared once within the src directory scope, and only needs to be declared once since it is done using the typescript keyword "declare global". The other time it is declared is in the development folder, so if we were to make a .d.ts, are you suggesting we put it in the global global scope outside of both src/ and development/?

Reply:
I'm suggesting we may be able to import it from a declaration file at the two points where we need it, and preserve the existing behavior / typing without extending the Node global anywhere else.

It might not be worth the trouble, though!